### DISCUSSION TITLE
Units of measurement for post-synaptic weights

### DISCUSSION THREAD

**Discussion by sarapiers** [jonescompneurolab/hnn-core#728] [Q&A] [answered]
Hello, 
I saw that on the alpha beta tutorial from the HNN Core and HNN GUI documentation that the synaptic weights for PN L2/3and L5 AMPA are given as 5.4 microS. On the Jones 2009 paper, which the tutorial should be referring to, it is stated: "The default conductances for FF and FB inputs were as follows: low mu, maximal weight of 0.4 picosiemens (pS) onto PNs and 0.8 pS onto INs; high mu, maximal weight of 0.6 pS onto PNs and 1.2 pS onto INs". Do they refer to the same thing?
If so, can you help me understand why there are two order of magnitude of difference between the parameter as given in the tutorial (5.4*1e-11 S) and the one given in the paper (4*1e-13 S for low mu and 6*1e-13 S for high mu)?

Thank you for your help,
Best,

Sara

**Comment by jasmainak** [comment]
cc-ing @stephanie-r-jones and @rythorpe ... probably the best people to answer this question

**Comment by rythorpe** [comment]
I'd trust the values in the tutorial/param files. I vaguely recall this coming up before and believe we determined the values in Jones et al. 2009 to be off by, as you say, two orders of magnitude.

**Reply by rythorpe** [reply, marked as answer]
The paper should say 4e-11, 8e-11, 6e-11, and 1.2e-10 siemens.

**Reply by sarapiers** [reply]
Ok, thank you for the clarification!

**Comment by katduecker** [comment]
On this note, are we using the default units from NEURON for the conductances (S/cm^2 for ion channels and µS for point processes?)

Would it be helpful to define our units somewhere in the code? In NEURON, this is solved by importing the units at the start of the script (they are just numbers) and then multiplying them with the values.
